### PR TITLE
RitikaG Coding_Exam_4_35mins

### PR DESCRIPTION
**Pull request by RitikaGOct2020**
image

image

### CODE REVIEW

File: src/ritika/exams/Coding_Exam_4_Pg1.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+output: 1 7
+*/
+public class Coding_Exam_4_Pg1 {
+
+	static void findCommonElementsBetweenArrays(int[] array1, int[] array2) {
+		if (array1.length == array2.length || array1.length < array2.length) {
+			System.out.println("Common elements:");
+			for (int index = 0; index < array1.length; index++) {
+				if (array1[index] == array2[index])
```
Comment (harshvegada) [inline src/ritika/exams/Coding_Exam_4_Pg1.java:17]: Hi Ritika,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly. I know this is not mentioned but this is like counter question on same program.

**Review by harshvegada — CHANGES_REQUESTED** [review]
One change require in array's program, second program code looks good to me!

File: src/ritika/exams/Coding_Exam_4_Pg1.java (hunk 1 above)
Comment (RitikaGOct2020) [reply src/ritika/exams/Coding_Exam_4_Pg1.java]: @harshvegada 
Required changes have been done and code is updated. Time taken to resolve issue is approx 15mins.
Thanks for reviewing!

**Review by harshvegada — APPROVED** [review]

**Comment by harshvegada** [comment]
Good to merge!

**Pull request merged**